ynr/apps/people/management/commands/import_person_identifiers.py

```diff
@@ -0,0 +1,65 @@
+import csv
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from candidates.models import PersonRedirect
+from people.models import PersonIdentifier, Person
+
+
+class Command(BaseCommand):
+    help = (
+        "Expects a two column CSV with the YNR ID as the first column "
+        "and the identifier to add as the second. There must be no column"
+        "headings in the CSV. This commands assumes a single ID of each type"
+        "and will overwrite IDs if there are duplicate for a person."
+    )
+
+    def add_arguments(self, parser):
+        parser.add_argument("--file", action="store", required=True)
+        parser.add_argument("--id-type", action="store", required=True)
+        parser.add_argument("--replace", action="store_true")
+
+    @transaction.atomic
+    def handle(self, *args, **options):
+        id_types = [
+            x[0] for x in PersonIdentifier.objects.select_choices() if x[0]
+        ]
+        if options["id_type"] not in id_types:
+            raise ValueError(
+                "{} isn't a valid ID type. Options are {}".format(
+                    options["id_type"], ", ".join(id_types)
+                )
+            )
+
+        in_file = csv.reader(open(options["file"]))
+        for line in in_file:
+            person_id = line[0]
+            person_qs = Person.objects.filter(id=person_id)
+            if not person_qs.exists():
+                redirect_qs = PersonRedirect.objects.filter(
+                    old_person_id=person_id
+                )
+                if redirect_qs.exists():
+                    person_id = redirect_qs.get().new_person_id
+                else:
+                    continue
```

Comment:
This is only a style point. This code looks like it will work just fine, but writing it with `.filter()` and `.exists()` feels slightly clunky. I'd probably tend to do something more like

```py
try:
    Person.objects.get(id=person_id)
except Person.DoesNotExist:
    try:
        person_id = PersonRedirect.objects.get(old_person_id=person_id).new_person_id
    except PersonRedirect.DoesNotExist:
        self.stderr.write("should we be logging this?")
        continue
```

(not tested, but you get the idea)

Also, this is a bit of a tangent, but it feels like in general this is a set of operations we're probably going to want to use more than once (in fact, we may already be doing this in several places). It might actually be useful to abstract to a helper in `person.models` something like:

```py
def get_person(person_id):
    try:
        return Person.objects.get(id=person_id)
    except Person.DoesNotExist:
        try:
            redirect_id = PersonRedirect.objects.get(old_person_id=person_id).new_person_id
            return Person.objects.get(id=redirect_id)
        except PersonRedirect.DoesNotExist:
            raise Person.DoesNotExist()
```

Happy to move that to an issue for another day if you want though.